mask_size_x: 5.0
mask_size_y: 5.0
mask_size_z: 5.0
